mobile/src/screens/ChatScreen.tsx

```diff
@@ -1,21 +1,51 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { Text, View, StyleSheet } from 'react-native';
+import { I18nContext } from '../context/I18nContext';
+import { TextStyles } from '../styles/TextStyles';
+import { Colors } from '../styles/Colors';
 
 /**
  * Left tab on navbar for chatting with volunteers
  */
 export const ChatScreen: React.FC = () => {
+  const i18nCtx = useContext(I18nContext);
   return (
     <View style={styles.container}>
-      <Text>Chat</Text>
+
+      <Text style={[TextStyles.h2, styles.title]}> {i18nCtx.t('letsTalk')}</Text> 
```

Comment:
In the mockup, Let's Talk definitely looks larger than the rest of the text - let's bump this up to h1